harvester/sources/factories/edurep_json/extraction.py

```diff
@@ -0,0 +1,60 @@
+import os
```

Comment:
This is now seen by git as a new file, while it is actually a mv. Stupid git. It needs commits after a rename, before editing or it gets confused about adding/editing/moving.